packages/realm-web/rollup.config.mjs

```diff
@@ -97,16 +104,28 @@ export default [
       },
     ],
     plugins: [
-      commonjs(),
-      typescript({
-        tsconfig: "src/dom/tsconfig.json",
+      // Providing an alias for the "bson" package
+      // See https://github.com/mongodb/js-bson/pull/669#pullrequestreview-1994786536 for more context
+      alias({
+        entries: [
+          {
+            find: "bson",
+            // We're assuming the location of a .cjs next to the .mjs bundle
+            replacement: require.resolve("bson").replace(".mjs", ".mjs"),
```

Comment:
I tried but failed to use `import.meta.resolve("bson/lib/bson.cjs")` but this fails since the library declare no sub-path exports:
> [!] Error: Package subpath './lib/bson.cjs' is not defined by "exports" in [...]/realm-js/node_modules/bson/package.json imported from [...]/realm-js/packages/realm-web/rollup.config.mjs